### PR TITLE
using proverWaitReadTimeout from config

### LINKED ISSUE #1081: TODO: make prover wait ready timeout configurable
## Rationale

The current Prover Wait Ready Timeout time is hardcoded

https://github.com/hermeznetwork/hermez-node/blob/3fc5b4c531394e9d6a9975e2fa798bbd5029e6a4/coordinator/pipeline.go#L25-L26

## Implementation

Create a new configuration field to receive this info and use the configuration instead of the hardcoded value.
The default value can continue to be the value hardcoded.

### PR DESCRIPTION
**Pull request by marioidival**
Closes #1081 

### What does this PR does?
Remove `proverWaitReadTimeout` constant from pipeline.go file and make it configurable

```Starting test
    ‚úì Init variables (8420ms)
    ‚úì Check ETH L1 deposit (48396ms)
    ‚úì Check ERC20 L1 deposit (48470ms)
    ‚úì Check ETH L1 force exit (48220ms)
    ‚úì Check ERC20 L1 force exit (34219ms)
    ‚úì Check ETH withdrawal from firsts forceExits (12299ms)
    ‚úì Check ERC20 withdrawal from firsts forceExits (12330ms)
    ‚úì Check ETH N L1 force exits (48333ms)
    ‚úì Check ETH withdrawal from previous N force exits (8132ms)
    ‚úì Check single ETH L2 transfer (30306ms)
    ‚úì Check single ERC20 L2 transfer (50279ms)
    ‚úì Transfer ETH to a non-existent Bjj address (40293ms)
    ‚úì Transfer ERC20 to a non-existent Bjj address (40243ms)
    ‚úì Transfer ETH to hermez ethereum address (30230ms)
    ‚úì Transfer ERC20 to hermez ethereum address (40245ms)
    ‚úì Transfer ETH to non-existent ETH address (40304ms)
    ‚úì Transfer ETH to hermez ethereum address from internal account (50272ms)
    ‚úì Transfer ERC20 to hermez ethereum address from internal account (40250ms)
    ‚úì Check multiple L2 ETH transfer in same batch (50777ms)
    ‚úì Check multiple L2 ETH transfer in different batches (171318ms)
    ‚úì Atomic txs (50472ms)
    ‚úì Check L2 ETH exit (40296ms)
    ‚úì Check L2 ERC20 exit (40281ms)
    ‚úì Check ETH withdrawal from L2 exit (8160ms)
    ‚úì Check ERC20 withdrawal from L2 exit (8135ms)
    ‚úì Exit ETH and circuit withdrawal (42597ms)
    ‚úì Assert API endpoints (1975ms)
    ‚úì Update Buckets (12251ms)
    ‚úì Check ETH L1 force exit bucket (44241ms)
    ‚úì Check ERC20 L1 force exit bucket (44233ms)
    ‚úì Check ETH withdrawal buckets (16286ms)
    ‚úì Check ERC20 withdrawal buckets (12259ms)
    ‚úì Check withdrawals buckets


  33 passing (20m)
```

### CODE REVIEW

**Review by jeffprestes — APPROVED** [review]

File: config/default.go [hunk 1]
```diff
@@ -54,6 +54,7 @@ PurgeByExtDelInterval = "1m"
 MustForgeAtSlotDeadline = true
 IgnoreSlotCommitment = true
 ForgeOncePerSlotIfTxs = false
+ProverWaitReadTimeout = "20s"
```
Comment (ARR552) [inline config/default.go:57]: Please add the new parameter to the documentation:
https://github.com/hermeznetwork/hermez-node/blob/develop/config/README.md

File: config/config.go [hunk 2]
```diff
@@ -125,6 +125,8 @@ type Coordinator struct {
 	// to the PurgeByExternalDelete function of the l2db which deletes
 	// pending txs externally marked by the column `external_delete`
 	PurgeByExtDelInterval Duration `validate:"required" env:"HEZNODE_COORDINATOR_PURGEBYEXTDELINTERVAL"`
+	// ProverWaitReadTimeout
+	ProverWaitReadTimeout Duration `env:"HEZNODE_COORDINATOR_PROVERWAITREADTIMEOUT"`
```
Comment (ARR552) [inline config/config.go:129]: Seems that this new parameter is required.
```suggestion
	ProverWaitReadTimeout Duration `validate:"required" env:"HEZNODE_COORDINATOR_PROVERWAITREADTIMEOUT"`
```

**Review by ARR552 — CHANGES_REQUESTED** [review]

File: config/config.go (hunk 2 above)
Comment (marioidival) [reply config/config.go]: @ARR552 thank you!

File: config/default.go (hunk 1 above)
Comment (marioidival) [reply config/default.go:57]: done!

File: config/README.md [hunk 3]
```diff
@@ -59,6 +59,7 @@ ForgeNoTxsDelay = HEZNODE_COORDINATOR_FORGENOTXSDELAY
 PurgeByExtDelInterval = HEZNODE_COORDINATOR_PURGEBYEXTDELINTERVAL
 MustForgeAtSlotDeadline = HEZNODE_COORDINATOR_MUSTFORGEATSLOTDEADLINE
 IgnoreSlotCommitment = HEZNODE_COORDINATOR_IGNORESLOTCOMMITMENT
+ProverWaitReadTimeout = "20s"
```
Comment (ARR552) [inline config/README.md:62]:
```suggestion
ProverWaitReadTimeout = HEZNODE_COORDINATOR_PROVERWAITREADTIMEOUT
```

Comment (ARR552) [reply config/README.md:62]: Also, you need to add it to the table with the param description. Line 173-174

**Review by ARR552 — CHANGES_REQUESTED** [review]

**Review by ARR552 — APPROVED** [review]

**Review by tclemos — APPROVED** [review]

**Pull request merged**